swarm/api/client/client_test.go

```diff
@@ -595,3 +604,41 @@ func TestClientCreateUpdateFeed(t *testing.T) {
 		t.Fatalf("Expected: %v, got %v", databytes, gotData)
 	}
 }
+
+func checkTag(t *testing.T, tags *chunk.Tags, split, stored, seen, total int) {
```

Comment:
once we have a way to query that tag, i'll do so. since the middleware creates the tag, without it being in the response there's no other way to do so